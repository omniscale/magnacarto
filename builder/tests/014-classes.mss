/* Applies to all layers with .land class */

#lakes {
  line-color: #f00;
  line-width: 0.5;
  polygon-fill: #0f0;
  // Applies to #lakes.land
  .land {
    polygon-fill: #00f;
  }
}



.water {
  polygon-fill: #0f0;
  line-width: 1;
  .basin {
    polygon-fill: #fff;
    polygon-opacity: 0.5;
  }
}
