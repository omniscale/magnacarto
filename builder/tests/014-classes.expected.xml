<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="lakes" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke="#ff0000" stroke-width="0.5"></LineSymbolizer>
      <PolygonSymbolizer fill="#00ff00"></PolygonSymbolizer>
    </Rule>
  </Style>
  <Layer name="lakes" srs="" status="off">
    <StyleName>lakes</StyleName>
  </Layer>
</Map>